packages/contentful--app-scripts/lib/create-app-definition/build-app-definition-settings.js

```diff
@@ -22,12 +22,12 @@ NOTE: This will create an app definition in your Contentful organization.
       message: `Select where your app can be rendered:`,
       type: 'checkbox',
       choices: [
-        { name: 'App configuration screen (app-config) ', value: 'app-config' },
-        { name: 'Entry field (entry-field) ', value: 'entry-field' },
-        { name: 'Entry sidebar (entry-sidebar) ', value: 'entry-sidebar' },
-        { name: 'Entry editor (entry-editor)', value: 'entry-editor' },
-        { name: 'Page (page) ', value: 'page' },
-        { name: 'Home (home) ', value: 'home' },
+        { name: 'App configuration screen ', value: 'app-config' },
```

Comment:
Why are your removing the internal names?

Reply:
Because we also removed them in web app. We also talked about it briefly with Lorenz and decided that we will remove them here.